src/physics/cam/physpkg.F90

```diff
@@ -1923,9 +1955,38 @@ subroutine tphysbc (ztodt, state,  &
          state,   ptend, cam_in%landfrac, pbuf)
 
     call physics_update(state, ptend, ztodt, tend)
+    ! NB: If we want to discard ZM tendencies here we need to call a ptend_dealloc
```

Comment:
Does that need to be added? Is this a new issue that should be raised? And what are "ZM tendencies"?